src/helpers/getDeckAdvice/advice/KINDREDS_GRACE/spec.js

```diff
@@ -1,7 +1,19 @@
 import advice from './'
 import getResolvedCardData from '../../../getResolvedCardData'
+import getRawCardData from '../../../getRawCardData'
 import serialisation from '../../../serialisation'
 import modifyDeck from '../../../modifyDeck'
+import { SPAWNS } from './index'
+
+const SPAWNS_ARRAY = Object.entries(SPAWNS)
```

Comment:
```suggestion
const SPAWNS_ARRAY = Object.keys(SPAWNS)
```
You don’t seem to be using the value in the loop, so picking the keys would be just as fine.